circle.yml

```diff
@@ -31,23 +27,12 @@ test:
     - gvm use stable && go version
     - gvm use stable && make build:
         pwd: $BASE_STABLE
+    - gvm use stable && GO15VENDOREXPERIMENT=1 go get github.com/dgageot/docker-machine-driver-none-external
```

Comment:
I know this is convenient, but can we find a way to do this without referencing a repository outside of this one? (e.g. using `cp -r` and `go install` from a subfolder of this repo)  If this repository needs to be updated it should be doable by anyone who has access to the Machine repo.